commands/explorepotatoes.js

```diff
@@ -216,7 +220,35 @@ const info = {
 					),
 					allowedMentions: { users: [] },
 				});
-			});
+			}
+			return _collector;
+		}
+
+
+		function _getMessages(board){
```

Comment:
I also moved this to a function because that `Promise.all` call was getting really messy